package.json

```diff
@@ -98,6 +100,7 @@
     "ngx-lottie": "^10.0.0",
     "ngx-matomo-client": "^6.0.2",
     "nouislider": "^15.7.1",
+    "ol": "^10.0.0",
```

Comment:
nit(non-blocking): nothing to do with you, but really wish openlayers just kept a package named openlayers! incredibly unhelpful when just skimming through deps list

Reply:
yes! And unhelpful when searching files for imports